src/ray/gcs/gcs_server/ray_syncer.h

```diff
@@ -53,7 +53,7 @@ class RaySyncer {
   void Start() {
     poller_->Start();
     broadcast_thread_ = std::make_unique<std::thread>([this]() {
-      SetThreadName("resource_report_broadcaster");
+      SetThreadName("resource_brdcst");
```

Comment:
broadcast -> bcast is more common, e.g., https://www.mpich.org/static/docs/v3.1/www3/MPI_Bcast.html

Reply:
got it.